src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/AssessmentTest.kt

```diff
@@ -1298,6 +1304,21 @@ class AssessmentTest : IntegrationTestBase() {
             AssessmentSortField.assessmentDueAt -> {
               ExpectedResponse.Error(HttpStatus.BAD_REQUEST, "Sorting by due date is not supported for CAS3")
             }
+
+            AssessmentSortField.applicationProbationDeliveryUnit -> {
+              ExpectedResponse.OK(
+                assessments
+                  .sortedWith(
+                    compareByDescending(nullsLast()) {
```

Comment:
When sorting a lists of strings that includes nulls, when using ORDER BY * DESC (via PageCriteria) Postgres puts nulls last, therefore `compareByDescending` and `nullsLast()` is required for this test to pass.

Reply:
It still the pdu in application assessment returned with null values line 1309
<img width="1599" alt="Screenshot 2024-08-20 at 15 59 12" src="https://github.com/user-attachments/assets/c6062193-48a7-4f88-929f-75ab39d7d92b">